### ISSUE TITLE
Inventory Keybinds Unfriendly to Lefties

### ISSUE THREAD

**Issue by kurazarrh** [anegostudios/VintageStory-Issues#977]
Can lefties get a bit of love from the gods of inventory? Shift-clicking stacks of items in the inventory (and possibly other inventory shortcuts that I'm not aware of... and which don't look to be documented anywhere) only works with the left shift key.

I'd like to request that the feature be expanded to "either" shift key, or be rebindable in settings so we lefties can use these features without having to move all over the keyboard.

I would request the same for the Esc key, since it can be used to quickly close all open inventories / windows. If I could rebind this to the Backspace key or something, it would be very helpful.

**Comment by Craluminum2413** [comment]
#404 ?

**Comment by kurazarrh** [comment]
@Craluminum2413 Not quite, I don't think. That bug is about unbinding keys; this is related to being able to rebind certain keys that currently cannot be rebound (or at least making the shift/ctrl/alt keys handedness-agnostic).

**Comment by Craluminum2413** [comment]
If I am not mistaken, you can already bind Shift/Ctrl/Alt keys to any other key

**Comment by kurazarrh** [comment]
You can bind Shift/Ctrl/Alt, yes, but you cannot bind the inventory function bound to Left Shift (which moves a full stack of items to/from inventories). There may be other inventory functions that I'm not aware of since I don't use the left side of the keyboard, but there is also new functionality coming with using Ctrl+mousewheel to pick up/drop individual quantities of items. I have concerns that lefties will be locked out of this feature, as well.

**Comment by kurazarrh** [comment]
Any updates on whether this bug will get some love soon? It would also be nice to be able to rebind the functionality that is currently on the Esc key (closing all open dialogs), as it fits the theme of accessibility for lefties.

**Comment by kurazarrh** [comment]
I'd just like to resurrect this old bug. I was kind of hoping to see this behavior resolved in 1.18 but it's still at large. Any chance it can get some love in the near future?

**Comment by kurazarrh** [comment]
@tyronx Can this bug get a little love? This issue has been a thorn in my side forever, and it's been buried since last time it was brought up.

**Comment by tyronx** [comment]
ok, next major update will allow right shift button for stack shifting. Esc key rebind is more complicated to do.
